### DISCUSSION TITLE
Would it run on Orange Pi 196 (arm cortex a5 32bit)?

### DISCUSSION THREAD

**Discussion by munecito** [tobychui/arozos#122] [Q&A] [answered]
I have a couple of these boards which I am not using at the moment and I was wondering if this would be a good project for them?

I am only asking before I start buying parts as I wouldn't want to get stuck with parts and no compatible SBC.

Thank you.

(EDIT: I think I found my own answer. That SBC has only 256mb of memory so I suppose it won't work)

**Comment by tobychui** [comment, marked as answer]
@munecito I don't think orange pi i96 even have a stable Linux kernel to begin with. 
So what SBCs you eventually pick up for ArozOS? 

-------------------------
Updates: orange pi i96 is discontinued and we do not have any plan to support it in the future.

**Reply by munecito** [reply]
Hello @tobychui .

I am revisiting your project as I am trying to get most of my files away from Google as it is getting expensive.

I picked up the orange pi 5 16GB. This one has a M2 port for an NVME drive but I am considering using a M2 key to convert that port to 4 or 5 SATA 3. 

I may use a case like the one you designed but with certain modifications as I would be powering SSD sata drives so no need for 12V but 5V instead.

Is ArozOS still alive?

**Reply by tobychui** [reply]
Hi @munecito ,

Cool! 

> Is ArozOS still alive?

Yes and no. We are working on the ArozOS v3 architecture internally (and we are still having issues on which distributed file system / dfs architecture we should be basing it on). According to our current roadmap, the v3 will looks identical to v2 to the user, but with dfs support built-in so you can use a few Pi NAS to build a cluster to max out a 2.5G or 10G network environment. And for v2 (current releases), we will still continue to fix bugs and do minor UX updates but it will got no new features any time soon.
